lib/engine/config/game/g_18_cz.rb

```diff
@@ -1559,7 +1559,7 @@ module G18CZ
                                       "nodes": [
                                           "town"
                                       ],
-                                      "pay": 0,
+                                      "pay": 3,
                                       "visit": 99
```

Comment:
Is this right? It seems like this would turn a 3E into a 3+3

(With the ability to skip towns, but it still gets paid for three towns with this config)

Reply:
This is manually overridden in the code. It runs the the top 3 locations. With this 3 it counts the towns as a stop. 
In the revenue_for all except the top 3 stops are removed

Reply:
Thanks I totally didn't grok that ha